src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseworkerDocumentManagement.java

```diff
@@ -78,7 +77,7 @@ private void doConfigure(final ConfigBuilder<CaseData, State, UserRole> configBu
             .showSummary()
             .grant(CREATE_READ_UPDATE,
                 SUPER_USER, ST_CIC_CASEWORKER, ST_CIC_SENIOR_CASEWORKER, ST_CIC_HEARING_CENTRE_ADMIN,
-                ST_CIC_HEARING_CENTRE_TEAM_LEADER, ST_CIC_SENIOR_JUDGE, ST_CIC_RESPONDENT)
+                ST_CIC_HEARING_CENTRE_TEAM_LEADER, ST_CIC_SENIOR_JUDGE)
             .grantHistoryOnly(SUPER_USER, ST_CIC_JUDGE)
```

Comment:
I can see SUPER_USER both in the CRU access and R access. Can we resolve this duplication.